catalog/ui/catalog-ui-search/src/main/webapp/js/router.js

```diff
@@ -119,6 +119,12 @@ define([
             hideViews();
             var self = this;
             var queryForMetacards, queryForMetacard;
+            var lowBandwidth = args.includes('lowBandwidth') || router.get('lowBandwidth');
```

Comment:
Since backbone blows away query params, it might be better to do this in the initialize method of the router.

Reply:
How will that affect the behavior if some hypothetical user routes to, say, `/search/catalog/#workspaces` and then decides "oh! I wanted low bandwidth mode!" and adds the URL parameter?

Reply:
As a user I would only expect it to work if it was there on first load.  However, we could add low bandwidth into the actual user settings to allow people to change it after load (maybe not in this PR but in a future PR).

The other half of this is that `includes` is not fully supported across all browsers.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
Is the query param even available when the router is initializing? Going off of the Backbone documentation, the only options it looks like it accepts on initialization are custom route hashes.

I set a breakpoint and navigated to Intrigue with the `?lowBandwidth` parameter, and it doesn't seem to even be accessible from that context unless I'm just not seeing something

Reply:
Refactored to look at `window.location.search` for the query parameter in the router's initialize function